src/RecipesBase.jl

```diff
@@ -104,6 +104,7 @@ function create_kw_body(func_signature::Expr)
             k, v = kwpair.args
             if isa(k, Expr) && k.head == :(::)
                 k = k.args[1]
+                warn("Type annotations on keyword arguments not currently supported. Type information has been discarded")
```

Comment:
@mkborregaard Is this what you had in mind? I can change the language of the warning if you think something else would be more informative.

Reply:
Yes exactly - thanks!